src/devices.js

```diff
@@ -307,7 +307,9 @@ function assembleInstallSteps(steps) {
                   if (
                     error &&
                     (error.includes("no device") ||
-                      error.includes("No such device"))
+                      error.includes("device offline") ||
```

Comment:
I'm no style expert, but isn't

```
error && (
  error.includes("no device") ||
  error.includes("No such device") ||
...
```

Preferable?

Reply:
This is what prettier.io did, i'd keep it this way.

Reply:
Okay